ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot

```diff
@@ -55,21 +55,41 @@ Run TestRayTuneHPOGpu ODH test with Python 3.11
     ...     WorkloadsOrchestration
     Run DistributedWorkloads ODH Test    TestMnistRayTuneHpoGpu    ${RAY_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
 
-Run TestKueueCustomRayCpu ODH test with Python 3.11
-    [Documentation]    Run Go ODH test: TestKueueCustomRayCpu
+Run TestKueueCustomRayCudaCpu ODH test with Python 3.11
+    [Documentation]    Run Go ODH test: TestKueueCustomRayCudaCpu
     [Tags]  RHOAIENG-10013
     ...     Tier1
     ...     DistributedWorkloads
     ...     Training
     ...     WorkloadsOrchestration
-    Run DistributedWorkloads ODH Test    TestMnistCustomRayImageCpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+    Run DistributedWorkloads ODH Test    TestMnistCustomRayCudaCpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
 
-Run TestKueueCustomRayGpu ODH test with Python 3.11
-    [Documentation]    Run Go ODH test: TestKueueCustomRayGpu
+Run TestKueueCustomRayCudaGpu ODH test with Python 3.11
+    [Documentation]    Run Go ODH test: TestKueueCustomRayCudaGpu
     [Tags]  RHOAIENG-10013
     ...     Resources-GPU    NVIDIA-GPUs
     ...     Tier1
     ...     DistributedWorkloads
     ...     Training
     ...     WorkloadsOrchestration
-    Run DistributedWorkloads ODH Test    TestMnistCustomRayImageGpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+    Run DistributedWorkloads ODH Test    TestMnistCustomRayCudaGpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+
+Run TestKueueCustomRayRocmCpu ODH test with Python 3.11
+    [Documentation]    Run Go ODH test: TestKueueCustomRayRocmCpu
+    [Tags]  RHOAIENG-12484
+    ...     ROCm
```

Comment:
Makes me think, this test runs on CPU so ROCm flag is not needed, right?

Reply:
Yes right. I will raise another PR to remove this tag

Reply:
Raised PR - https://github.com/red-hat-data-services/ods-ci/pull/2127